src/ocamlorg_web/lib/handler.ml

```diff
@@ -414,6 +414,7 @@ let package_versioned t kind req =
         | Some doc -> (doc, "README")
         | None -> (description |> Omd.of_string |> Omd.to_html, "Description")
       in
+      let* changes_file = Ocamlorg_package.changes_file ~kind package in
       let _license = Ocamlorg_package.license_file ~kind package in
```

Comment:
We're not passing the license file here, we could do that to check wether we were able to find it in the doc and link to it depending on that.